coverage-tests/overrides.js

```diff
@@ -0,0 +1,90 @@
+    'check regions by coordinates in frame with scroll stitching': {skip: true},	//Unable to locate element: {"method":"css selector","selector":"#modal2"}
+    'check regions by coordinates in frame with vg': {skip: true},			//diffs
+    'should find regions by visual locator': {skip: true},				//Visual locators not implemented
+    'should find regions by visual locator with vg': {skip: true},			//Visual locators not implemented
+    'should not check if disabled': {skip: true},					//Unable to locate element: {"method":"css selector","selector":"[id="someId"]"}
+    'should hide and restore scrollbars with scroll stitching': {skip: true},		//diff
+    'should hide and restore scrollbars with vg': {skip: true},				// Unable to locate element: {"method":"css selector","selector":"#inner-frame-div"}
+    'should send floating region by coordinates in frame with vg': {skip: true},	//diffs
+    'should not send dom': {skip: true},						//diffs - info["actualAppOutput"][0]["image"]["hasDom"] == False,    assert True == False
+    'should send dom and location when check window with vg': {skip: true}, // not implemented in VG
+    'should send dom on ie': {skip: true},						//diffs - assert info["actualAppOutput"][0]["image"]["hasDom"] == True   assert False == True
+    'should send dom on edge legacy': {skip: true},					//eyes.open(driver)   -  EyesError: eyes.open_base() failed
+    'should set viewport size on edge legacy': {skip: true},				//EyesError: Failed to set the viewport size
+    'check window after manual scroll on safari 11': {skip: true},			//diffs
+    'check window after manual scroll with vg': {skip: true},				//diffs
+    'should send accessibility regions by selector with css stitching': {skip: true},		//Index error. [imageMatchSettings][accessibility] is empty...
+    'should send accessibility regions by selector with scroll stitching': {skip: true},	//Index error. [imageMatchSettings][accessibility] is empty...
+    'check window fully on android chrome emulator on desktop page': {skip: true},		//diffs
+    'check window fully on android chrome emulator on mobile page with horizontal scroll': {skip: true},	//diffs
+    'check window fully on android chrome emulator on mobile page': {skip: true},		//diffs
+    'check window with layout breakpoints': {skip: true},				//layout breakpoints are not implemented
+    'check window with layout breakpoints in config': {skip: true},			//layout breakpoints are not implemented
+    'check window on page with sticky header with vg': {skip: true},			//diffs
+    'check window fully with custom scroll root with css stitching': {skip: true},	//diffs
+    'check window fully with fixed scroll root element': {skip: true},			//diffs
+    'check scrollable modal region by selector fully with scroll stitching': {skip: true},	//diffs
+    'check window fully and frame in frame fully with vg': {skip: true},		//diffs
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7812239)
```suggestion
    "check window fully and frame in frame fully with vg": {skip: true},		//diffs
```